src/plans/create/steps/review/MigrationTypeReviewSection.tsx

```diff
@@ -13,31 +13,47 @@ import { useForkliftTranslation } from '@utils/i18n';
 
 import { planStepNames, PlanWizardStepId } from '../../constants';
 import { useCreatePlanFormContext } from '../../hooks/useCreatePlanFormContext';
+import { hasLiveMigrationProviderType } from '../../utils/hasLiveMigrationProviderType';
+import { hasWarmMigrationProviderType } from '../../utils/hasWarmMigrationProviderType';
+import { GeneralFormFieldId } from '../general-information/constants';
 import { MigrationTypeFieldId, migrationTypeLabels } from '../migration-type/constants';
 
-const MigrationTypeReviewSection: FC = () => {
+const MigrationTypeReviewSection: FC<{ isLiveMigrationEnabled: boolean }> = ({
+  isLiveMigrationEnabled,
+}) => {
   const { t } = useForkliftTranslation();
   const { goToStepById } = useWizardContext();
   const { control } = useCreatePlanFormContext();
-  const migrationType = useWatch({ control, name: MigrationTypeFieldId.MigrationType });
+  const [migrationType, sourceProvider] = useWatch({
+    control,
+    name: [MigrationTypeFieldId.MigrationType, GeneralFormFieldId.SourceProvider],
+  });
+
+  const isVisible =
+    hasWarmMigrationProviderType(sourceProvider) ||
+    (hasLiveMigrationProviderType(sourceProvider) && isLiveMigrationEnabled);
 
   return (
-    <ExpandableReviewSection
-      title={planStepNames[PlanWizardStepId.MigrationType]}
-      testId="review-migration-type-section"
-      onEditClick={() => {
-        goToStepById(PlanWizardStepId.MigrationType);
-      }}
-    >
-      <DescriptionList isHorizontal horizontalTermWidthModifier={{ default: '18ch' }}>
-        <DescriptionListGroup>
-          <DescriptionListTerm>{t('Migration type')}</DescriptionListTerm>
-          <DescriptionListDescription data-testid="review-migration-type">
-            {migrationTypeLabels[migrationType]}
-          </DescriptionListDescription>
-        </DescriptionListGroup>
-      </DescriptionList>
-    </ExpandableReviewSection>
+    <>
+      {isVisible && (
```

Comment:
I think an early return here would be clearer than returning an empty JSX tag. Also, the const `isVisible` should read differently so we only have to read the const name itself instead of reading the expression to find out why its not visible.

something like 

```
if (!providerSupportsMigrationTypes) {
  return;
}

return (...the expandable section);
```